example/src/App.tsx

```diff
@@ -48,22 +51,19 @@ export default function App() {
     DevSettings.addMenuItem('Show Native Stack Example', () => {
       setCurrentExample('native-stack');
     });
-    DevSettings.addMenuItem('Show React Native Safe Area View Example', () => {
-      setCurrentExample('react-native-safe-area-view');
-    });
   }, []);
 
   switch (currentExample) {
-    case 'simple':
-      return <SimpleExample />;
+    case 'safe-area-view':
+      return <SafeAreaViewExample />;
+    case 'hooks':
+      return <HooksExample />;
     case 'react-navigation-4':
       return <ReactNavigation4Example />;
     case 'react-navigation-5':
       return <ReactNavigation5Example />;
     case 'native-stack':
       return <NativeStackExample />;
-    case 'react-native-safe-area-view':
```

Comment:
Let's leave this example, I added to avoid breaking react-native-safe-area-view since it uses this library.